webview/src/experiments/components/table/styles.module.scss

```diff
@@ -590,70 +468,76 @@ th {
       background-color: $row-border-selected-color;
     }
   }
+}
 
-  &:hover:not(.rowSelected) {
-    td:not(.experimentCell),
-    .experimentCell::before {
-      background-color: $row-hover-background-color;
-    }
+.nestedRow {
+  .experimentCell .innerCell {
+    padding-left: calc($nested-row-padding + $edge-padding);
 
-    td:not(.experimentCell):hover::before,
-    td:hover + td::before {
-      background-color: $border-color;
+    .rowActions {
+      left: calc(($cell-padding + $nested-row-padding) * -1);
     }
   }
 }
 
-.workspaceRowGroup {
-  border: none;
-  position: sticky;
-  top: var(--table-head-height);
-  z-index: 4;
-  background-color: $row-bg-color;
-
-  &.withShadow {
-    box-shadow: 0 5px 8px -2px $shadow;
+.previousCommitsRow {
+  border-bottom: $row-border;
 
-    tr {
-      border-bottom: none;
-    }
+  td {
+    font-size: 0.6rem;
+    padding-left: 16px;
   }
+}
 
-  .innerCell {
-    padding: 5px 10px;
```

Comment:
I noticed that we have the workspace cells slightly misaligned with the other rows. I assumed it wasn't intentional and deleted the line.